lib/engines/content_object_store/test/functional/sentry_tags_test.rb

```diff
@@ -1,28 +1,32 @@
 require "test_helper"
 require "sentry/test_helper"
 
-class ContentObjectStore::SentryTagsTest < ActionDispatch::IntegrationTest
+class ContentObjectStore::SentryTagsTest < ActionController::TestCase
+  tests ContentObjectStore::ContentBlock::DocumentsController
   include Sentry::TestHelper
 
   setup do
     setup_sentry_test
     feature_flags.switch!(:content_object_store, true)
     login_as_admin
+    @routes = ContentObjectStore::Engine.routes
   end
 
   teardown do
     teardown_sentry_test
   end
 
   test "throwing an error includes tags" do
-    exception = ArgumentError.new("Cannot find schema for block_type")
+    exception = ArgumentError.new("Some error")
     raises_exception = ->(*_args) { raise exception }
 
     ContentObjectStore::ContentBlock::Document.stub :all, raises_exception do
-      get content_object_store.content_object_store_content_block_documents_path
-    rescue ArgumentError
-      assert_equal sentry_events.count, 1
-      assert_equal sentry_events[0].tags[:engine], "content_object_store"
+      get :index
+    rescue ArgumentError => e
+      Sentry.capture_exception(e)
     end
+
+    assert_equal sentry_events.count, 1
```

Comment:
Noticed when testing this that we probably want these the other way around so [the expected is first and the actual is second](https://rubydoc.info/gems/minitest/Minitest/Assertions#assert_equal-instance_method). Otherwise it reads backwards on failure:

```
Expected: nil
  Actual: "content_object_store"
```